test/commands/share.js

```diff
@@ -0,0 +1,39 @@
+import sinon from 'sinon';
+
+import * as add from '../../src/scripts/add';
+import * as addAll from '../../src/scripts/add-all';
+import * as push from '../../src/scripts/push';
+import * as runWithTruffle from '../../src/utils/runWithTruffle'
+import Session from '../../src/models/network/Session'
+import program from '../../src/bin/program';
+
+exports.setup = function () {
+  beforeEach('set up stubs', function () {
+    this.add = sinon.stub(add, 'default')
+    this.addAll = sinon.stub(addAll, 'default')
+    this.push = sinon.stub(push, 'default')
```

Comment:
Later, we'll need to stub all commands here. Note that you can access a list of all commands from `src/commands/index` and loop over it creating the stubs. However, I'm not sure if the stub will work correctly when imported from that list, but it may be worth a shot.

Reply:
actually we stub `src/scripts/...` here so there're no `index` file. Even if we create an `index` for `scripts` we till need to change all `import` in `commands` to make it stubbale